src/components/look-controls.js

```diff
@@ -20,6 +20,15 @@ module.exports.Component = registerComponent('look-controls', {
     this.setupMouseControls();
     this.setupHMDControls();
     this.bindMethods();
+
+    // Enable grab cursor class on canvas.
+    var sceneEl = this.el.sceneEl;
```

Comment:
If we are worried about duplicate definitions, linters will catch that instantly.

I am not worried about scoping. Been following this style for months, I have been bothered and felt bogged down due to increased maintenance. I detailed the examples in #1679, every little change requires more code in different places while downgrading readability.